index.html

```diff
@@ -746,6 +746,148 @@ <h3>JcsEd25519Key2020</h3>
     </pre>
   </section>
 
+</section>
+
+<section>
+  <h2>BBS+ Signature 2020</h2>
+  
+  <section>
+
+  <h3>BbsBlsSignature2020</h3>
+
+  <table class="simple">
+    <thead>
+      <tr>
+        <th colspan=2>Summary</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <tr>
+        <td>Identifiers</td>
+        <td>BbsBlsSignature2020</td>
+      </tr>
+      <tr>
+        <td>Status</td>
+        <td>PROVISIONAL</td>
+      </tr>
+      <tr>
+        <td>Authors</td>
+        <td>Tobias Looker, Orie Steele</td>
+      </tr>
+      <tr>
+        <td>Specification</td>
+        <td><a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Signatures 2020</a></td>
+      </tr>
+    </tbody>
+  </table>
+
+  <h3>BbsBlsSignatureProof2020</h3>
+
+  <table class="simple">
+    <thead>
+      <tr>
+        <th colspan=2>Summary</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <tr>
+        <td>Identifiers</td>
+        <td>BbsBlsSignatureProof2020</td>
+      </tr>
+      <tr>
+        <td>Status</td>
+        <td>PROVISIONAL</td>
+      </tr>
+      <tr>
+        <td>Authors</td>
+        <td>Tobias Looker, Orie Steele</td>
+      </tr>
+      <tr>
+        <td>Specification</td>
+        <td><a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Signatures 2020</a></td>
+      </tr>
+    </tbody>
+  </table>
+
+  <h3>Bls12381G1Key2020</h3>
```

Comment:
This is a Verification key, so should be "Bls12381G1VerificationKey2020"... if we don't do that, people will use it to store private keys, and we don't want that.

Reply:
The word "Verification" does not prevent someone from storing private keys... the solution to this problem is spec text, not bike shedding. 

I suggest we ad text to `ld-proofs` `ld-cryptosuite-registry`, and not try and force a naming convention on contributors to this registry which provides nothing but security theatre.

Reply:
If the term "VerificationKey" is required in every string name for a key that is meant to be used for verification, it should say so in the ld-proof spec.... which i would object too, based on the naming convention being less important than normative spec text / guidance....

also... the term "VerificationKey" occurs exactly once in the ld proofs spec, here: https://w3c-ccg.github.io/ld-proofs/#example-7

and its for "Ed25519VerificationKey2018"...which is not correct, since technically the underlying curve is "Curve25519" and its used with Ed25519 Signature Scheme and X25519 Key Agreement via ECDH... So the correct thing would have been to name things like this:

`Curve25519Base58BtcVerificationKey2018` `Ed25519Base58BtcVerificationKey2018` `Ed25519DetatchedJsonWebSignature2018`

All of which would line up with libsodium much better... except that lib sodium DOES NOT USE the term "VerificationKey" anywhere!
 
https://doc.libsodium.org/?q=Verification%20Key

Nor does the term appear here: https://en.wikipedia.org/wiki/Twisted_Edwards_curve

Googling "Verification Key" this seems like the most "normative" definition...

https://www.pcmag.com/encyclopedia/term/verification-key

If we are going to require people to use "VerificationKey" in all their ld suites... it should say:

`Verification Key is a synonym for public key, used to describe the key used to verify a proof`

`VerificationKey MUST be included in the `type` of all linked data proof verification methods`.

In absence of such language, its ridiculous to tell people they must include 15 extra bytes.

Reply:
> In absence of such language, its ridiculous to tell people they must include 15 extra bytes.

I'd be happy to pull in a PR to that effect. I do think we should say that somewhere in the specs... probably the LD Signature spec.

To be clear Verification Key *is not* a synonym for public key... a public key can be used for (at least) both verification of signatures and key agreement (along with more specific cryptographic protocols like PAKE, Pallier Homomorphic encryption and others, which one could argue should get their own key type for purpose).